lib/plugins/aws/invokeLocal/fixture/asyncHandlerWithSuccess.js

```diff
@@ -1,51 +1,50 @@
 'use strict';
 
-module.exports.withError = () => Promise.reject(new Error('failed'));
```

Comment:
Did you verify that these are correctly used if they no longer return a promise? The name of the fixture `asyncHandlerWithSuccess` suggests that it's expected that these handlers are asynchronous

Reply:
I thought about that, but I don't think there's a logical difference between returning a `Promise.reject()` vs an `Error`. If you look at all the places where this script is used, it awaits event the functions that weren't defined as promise returning functions. It's totally your call, but I think returning the error is okay.